com.obsproject.Studio.Plugin.NDI.metainfo.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>com.obsproject.Studio.Plugin.NDI</id>
+  <extends>com.obsproject.Studio</extends>
+  <name>obs-ndi</name>
+  <summary>NewTek NDI integration for OBS Studio</summary>
+  <url type="homepage">https://github.com/obs-ndi/obs-ndi</url>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
```

Comment:
Given that this uses extra_data to download something the seems to require an EULA, I doubt this is the correct license.

Reply:
The Project in under GPLv2 and the used in the project NDI header are under MIT.

But I guess the NDI Agreement takes over when it comes to the metainfo.

Reply:
but what is downloaded by the extra-data and why?

Reply:
The NDI library, which is required by obs-ndi to work.

Reply:
Which is not FOSS.

Reply:
Already fixed

Reply:
Just to specify, I considered that `project_license` asked for the license of project itself not with all it's dependency took in account.